cmd/root.go

```diff
@@ -48,9 +48,10 @@ type RootCommandConfig struct {
 	UnsupportedRepos []string
 
 	// package scoped, these are mostly for caching
-	setupConfigRun bool
-	imagePulled    map[string]bool
-	cachedEnvVars  map[string]string
+	setupConfigRun     bool
+	imagePulled        map[string]bool
+	cachedEnvVars      map[string]string
+	cachedDockerLabels map[string]string
```

Comment:
Maybe we should call these `cachedStackLabels`? The term Docker is too ambiguous